src/resources/postcss/common-responsive.pcss

```diff
@@ -0,0 +1,232 @@
+
+	/* -------------------------------------------------------------------------
+	 * Heading: h1
+	 * ------------------------------------------------------------------------- */
+
+	 .tribe-common-h1 {
+
+		@media(--viewport-medium) {
+			@mixin desktop-heading-1;
+		}
+	}
+
+	/* -------------------------------------------------------------------------
+	 * Heading: h2
+	 * ------------------------------------------------------------------------- */
+
+	.tribe-common-h2 {
+
+		@media(--viewport-medium) {
+			@mixin desktop-heading-2;
+		}
+	}
+
+	/* -------------------------------------------------------------------------
+	 * Heading: h3
+	 * ------------------------------------------------------------------------- */
+
+	.tribe-common-h3 {
+
+		@media(--viewport-medium) {
+			@mixin desktop-heading-3;
+		}
+	}
+
+	/* -------------------------------------------------------------------------
+	 * Heading: h4
+	 * ------------------------------------------------------------------------- */
+
+	.tribe-common-h4 {
+
+		@media(--viewport-medium) {
+			@mixin desktop-heading-4;
+		}
+	}
+
+	/* -------------------------------------------------------------------------
+	 * Heading: h6
+	 * ------------------------------------------------------------------------- */
+
+	.tribe-common-h6 {
+
+		@media(--viewport-medium) {
+			@mixin desktop-heading-6;
+		}
+	}
+
+	/* -------------------------------------------------------------------------
+	 * Heading: h3 for --viewport-medium
+	 * ------------------------------------------------------------------------- */
+
+	.tribe-common-h3--min-medium {
+
+		@media(--viewport-medium) {
+			@mixin desktop-heading-3;
+		}
+	}
+
+	/* -------------------------------------------------------------------------
+	 * Heading: h4 for --viewport-medium
+	 * ------------------------------------------------------------------------- */
+
+	.tribe-common-h4--min-medium {
+
+		@media(--viewport-medium) {
+			@mixin desktop-heading-4;
+		}
+	}
+
+	/* -------------------------------------------------------------------------
+	 * Heading: h5 for --viewport-medium
+	 * ------------------------------------------------------------------------- */
+
+	.tribe-common-h5--min-medium {
+
+		@media(--viewport-medium) {
+			@mixin heading-5;
+		}
+	}
+
+	/* -------------------------------------------------------------------------
+	 * Heading: h6 for --viewport-medium
+	 * ------------------------------------------------------------------------- */
+
+	.tribe-common-h6--min-medium {
+
+		@media(--viewport-medium) {
+			@mixin desktop-heading-6;
+		}
+	}
+
+	/* -------------------------------------------------------------------------
+	 * Heading: h7 for --viewport-medium
+	 * ------------------------------------------------------------------------- */
+
+	.tribe-common-h7--min-medium {
+
+		@media(--viewport-medium) {
+			@mixin heading-7;
+		}
+	}
+
+	/* From: base/skeleton/forms/_text.pcss */
```

Comment:
Right now there's no skeleton vs full. It's a bit tricky because that setting is added when used in combination of TEC. I feel that if we want to handle skeleton/full we would need to review it in detail, and the implications that may bring (challenges for users already using stuff, different settings combinations, plugins active, etc).

Reply:
Sounds like another level to this. No worries, thought I'd ask in case.